CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/ValueSerializer.java

```diff
@@ -46,7 +47,8 @@ else if (value instanceof Map) {
                 String string = serialize(entry.getValue(), memory);
                 stringMap.put(key, string);
             }
-            return new Gson().toJson(stringMap);
+            Gson gson = new GsonBuilder().serializeNulls().create();
```

Comment:
wouldn't this be better:
return new GsonBuilder().serializeNulls().create().toJson(stringMap);
??